### PR TITLE
my user3 file

### PR DESCRIPTION
**Pull request by Dzuiest**

### CODE REVIEW

**Comment by scottreed93** [comment]
I hope you get your coffee ;)

**Comment by UnCommitted** [comment]
Nice work! We'll merge now

File: user3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+I am desperate for a strong coffee :)
```
Comment (UnCommitted) [inline user3.txt:1]: I prefer tea though

**Comment by jatin-agarwal** [comment]
get back to work. where is ur code. pull request rejected.

**Pull request merged**